joss/paper.md

```diff
@@ -0,0 +1,88 @@
+
+## Statement of need
+
+Synthetic data, with a known generating process, is required in many scenarios, including teaching concepts on data collection, data cleaning and data visualisation; and testing analysis methods for precision, adequacy and robustness. In the analysis of infectious disease outbreaks, termed _outbreak analytics_ [@polonskyOutbreakAnalyticsDeveloping2019a; jombartWhyDevelopmentOutbreak2021; @carterHowImproveOutbreak2021], one of the most common forms of data is a line list. This is a tabular dataset where each row corresponds to a single case in the outbreak, recording information on person, place and time. Information for each case can include, but is not limited to, personal information (name, age, sex), key event dates (date of symptom onset, date of hospitalisation, date of outcome), clinical outcomes, and geographic information (coordinates, administrative area). Understanding and characterising the transmissibility and severity of infectious disease outbreaks typically use line list data [@coriKeyDataOutbreak2017]. 
+
+In the ecosystem of R packages for epidemiology, there are a range of tools for simulating and analysing line list data [@jombartCRANTaskView2025], however, there are no stable, actively maintained and tested R packages that simulate epidemiologically realistic outbreak data[^1]. The simulist R package offers functionality to simulate outbreak data (line list and contact tracing) with options for users to define the epidemiological and demographic characteristics of the outbreak, while also being designed on the principle of maximum interoperability with other related R packages, e.g. {epicontacts} [@campbellEpicontactsHandlingVisualisation2017], {epiparameter} [@lambertEpiparameterClassesHelper2025], and {cleanepi} [@maneCleanepiCleanStandardize2024]. Accepting epidemiological parameters directly from {epiparameter} facilitates generating synthetic data with the characteristics of past outbreaks.
+
+## State of the field
+
+The majority of outbreak analytics uses R and R packages extending the language to answer common epidemiological questions in an outbreak (ref). From basic data wrangling and data science using tools such as the tidyverse [@wickhamWelcomeTidyverse2019], to using advanced methods, for example nowcasting/forecasting of transmissibility [@abbottEstimatingTimevaryingReproduction2020a]  This is accompanied by several initiatives that teach epidemiology using R tooling [@batraEpidemiologistHandbook2021; @vallecamposEpiverseTRACETutorials2025] resulting in an active ecosystem of open-source tooling, teaching, and community engagement [@morganWHOHubPandemic2022], for example the [epinowcast community](https://www.epinowcast.org/).
```

Comment:
To add (ref)?

Reply:
This was a placeholder as I wasn't sure what was the best citation. I've gone with citing the 100 days paper, as although it doesn't explicitly talk about the amount of epidemiology implemented in R, it does talk about a lot of different epi initiatives that use R.

If there is a better or addition sources to cite please do let me know.